pasqal_cloud/__init__.py

```diff
@@ -298,22 +305,68 @@ def get_job(self, id: str, wait: bool = False) -> Job:
                 job_rsp = self._get_job(id)
         return Job(**job_rsp, _client=self._client)
 
+    def add_jobs(self, batch_id: str, jobs: List[CreateJob]) -> Batch:
```

Comment:
These two methods are duplicates of methods from the `batch` class; I understand we want to expose them at the top level for easy access in pasqal cloud but this makes me wonder a few things:
- a little duplication is fine but maybe we could manage to use these new functions in the Batch method to avoid the the duplication of the error catching
- Should we ditch entirely the batch class methods and make it a simple dataclass with no methods? I don't see much benefits for them at this point. For backwards compat, we probably have to stick with them 
- We really need to rename on the backend side the "complete" flag to "open"

Reply:
Given the terminology we use absolutely everywhere is 'open batch', changing the references from `complete` to `open` makes sense. Otherwise we should stop calling them open batches and rather noncomplete batches. I'd like it if we can avoid more rebatch-esque naming scenarios.

It does not look like we're using the methods bound to the Batch model, so I'd be fine with removing them.

Regarding to backwards compatibility, we can leave things for now but would be nice to know we're able to depreciate things eventually. Not much benefit providing the same interface in two places because someone doesn't want to update their script.